server/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -22,6 +22,7 @@ grant codeBase "${codebase.elasticsearch-secure-sm}" {
 grant codeBase "${codebase.elasticsearch}" {
   // needed for loading plugins which may expect the context class loader to be set
   permission java.lang.RuntimePermission "setContextClassLoader";
+  permission java.net.SocketPermission "*", "resolve, listen, accept";
```

Comment:
I was wondering if I should restrict this to 127.0.0.1, but I don't know if it will work properly for IPv6. (::1)

Reply:
The code is currently opening the server socket as follows:
   `ServerSocketChannel.open(StandardProtocolFamily.INET);`
, so is creating an IPv4-only socket. so restricting to an IPv4 loopback literal would be ok here, but not strictly needed.

Maybe the intention is really to open the server socket channel withOUT the protocol family specified, therefore supporting both IPv4 and IPv6, as configured on the local system.